XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -109,99 +171,324 @@
+    
+&lt;!-- DataTemplate used: --&gt;
+
+&lt;DataTemplate x:Key="ContactListViewTemplate" x:DataType="local1:Contact">
+    &lt;Grid&gt;
+        &lt;Grid.RowDefinitions&gt;
+            &lt;RowDefinition Height="*"/&gt;
+            &lt;RowDefinition Height="*"/&gt;
+        &lt;/Grid.RowDefinitions&gt;
+        &lt;Grid.ColumnDefinitions&gt;
+            &lt;ColumnDefinition Width="Auto"/&gt;
+            &lt;ColumnDefinition Width="*"/&gt;
+        &lt;/Grid.ColumnDefinitions&gt;
+        &lt;Ellipse x:Name="Ellipse"
+                    Grid.RowSpan="2"
+                    Width ="32"
+                    Height="32"
+                    Margin="6"
+                    VerticalAlignment="Center"
+                    HorizontalAlignment="Center"
+                    Fill="Gray"/&gt;
+        &lt;TextBlock Grid.Column="1"
+                    Text="{x:Bind Name}"
+                    x:Phase="1"
+                    Style="{ThemeResource BaseTextBlockStyle}"
+                    Margin="12,6,0,0"/&gt;
+        &lt;TextBlock  Grid.Column="1"
+                    Grid.Row="1"
+                    Text="{x:Bind Company}"
+                    x:Phase="2"
+                    Style="{ThemeResource BodyTextBlockStyle}"
+                    Margin="12,0,0,6"/&gt;
+    &lt;/Grid>
+&lt;/DataTemplate&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="SelectionMode" Value="{x:Bind Control2.SelectionMode, Mode=OneWay}"/>
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample HeaderText="ListView with Drag, Drop, and Reordering Support">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="1*"></RowDefinition>
+                    <RowDefinition Height="4*"></RowDefinition>
+                </Grid.RowDefinitions>
+                <TextBlock Grid.Row="0" TextWrapping="Wrap">
+                    In this ListView, you can drag and drop within a list to reorder it, or drag and drop between both lists to move and reorder items.
+                </TextBlock>
+                <StackPanel Orientation="Horizontal" Grid.Row="1">
+                <ListView
+                    x:Name="DragDropListView"
+                    ItemTemplate="{StaticResource ContactListViewTemplate}"
+                    CanDragItems="True"
+                    CanReorderItems="True"
+                    AllowDrop="True"
+                    SelectionMode="Single" 
+                    BorderThickness="1"
+                    BorderBrush="DimGray"
+                    MinWidth="350" 
+                    Height="400"
+                    DragItemsStarting="Source_DragItemsStarting"
+                    DragOver="Source_DragOver"
+                    Drop="Source_Drop"
+                    Margin="15"
+                    />
```

Comment:
nit: it's a stylistic choice, but I really hate the hanging closing bracket on its own line.